volume/local/local_unix.go

```diff
@@ -164,7 +164,7 @@ func (v *localVolume) unmount() error {
 }
 
 func (v *localVolume) CreatedAt() (time.Time, error) {
-	fileInfo, err := os.Stat(v.path)
+	fileInfo, err := os.Stat(v.rootPath)
```

Comment:
Oh, interesting; not sure why I didn't pick `rootPath` at the time; perhaps I wasn't sure if "rootPath" could have a different meaning. 🤔

Reply:
OH! 😂 I added it; https://github.com/moby/moby/commit/d3930330a797b1c1c02b6c937eee353154102a3a (https://github.com/moby/moby/pull/43632)